elearning/going_online/create_your_course.rst

```diff
@@ -0,0 +1,195 @@
+
+The course responsible can be changed by clicking the arrow next to their name at any moment.
+If you have multiple websites you can choose which website your course is going to be displayed on.
+
+Access Rights
+~~~~~~~~~~~~~
+
+Access rights will allow you to decide who can access your courses. If you choose **Public**, anyone who finds the
+course on your website will be able to enroll and access your content.
+
+.. image:: media/enroll-policy.png
+   :align: center
+   :alt: picking the enroll policy
+
+You can send invite mails directly from the dashboard of the eLearning App.
+Go to the settings button of the course you want to invite people to and click on invite.
+This will allow you to send a mail inviting people to your course.
+
+.. image:: media/invite.png
+   :align: center
+   :alt: invite students to your course
+
+When choosing the **Enroll Policy** **On Invitation** a smart **invite** button will appear on the course. This will
+allow you to send mails to students so they can enroll.
+
+.. image:: media/share.png
+   :align: center
+   :alt: sharing your course
+
+You can also invite people from the course on the website with the share button who will provide you with a link to
+access the course, you can then copy the text and send it to whoever you want to invite to the course. You also have
+the possibility to directly share the course on **Social Networks** with our smart buttons.
+
+.. image:: media/social.png
+   :align: center
+   :alt: sharing the course on social networks
+
+Display
+~~~~~~~
+
+If the course is public, everyone will be able to see and access the course. When the display is set to members only,
+you have to be an attendee of the course to see the contents and enroll in the course.
+You can also choose which content will be shown first in your course.
+**Latests Published** new content will always appear first.
+**Most Voted** the content that you audience liked the most will appear as featured.
+**Most Viewed** the content that has the most views will appear as featured.
+
+.. image:: media/display.png
+   :align: center
+   :alt: picking the display for your course
+
+Here is an example where you can see the featured content above the rest of the course content.
+
+.. image:: media/disp.png
+   :align: center
+   :alt: picking the display for your course
+
+.. note::
+   This will only affect the display if you have chosen Documentation as course type.
+
+Description
+-----------
+
+A course description is a brief summary of the content that’s in your course. Course descriptions appear on the Website
+along the title of the course. They can be very useful for students to understand quickly what the course is about.
+
+.. image:: media/description.png
+   :align: center
+   :alt: course description for your course
+
+The course description will appear under the title of your course on the website and once the course is published
+everyone enrolling will have access to the description.
```

Comment:
this is said in the paragraph above!?
Idea: delete the first image and second paragraph, and keep the first paragraph with the second image.